__tests__/integrations/php.Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM alpine
```

Comment:
Originally tried using a `8.1.8-fpm` image instead of `apline` but I ran into a bunch of problems trying to get Node installed that I didn't feel like spending a bunch of time to sort out. Only real downside here is that the image is bigger than it needs to be, and installing these PHP deps takes a little longer but 🤷

Reply:
Agreed. I have zero interest in writing and maintaining a Bash test suite.